src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/view.html.twig

```diff
@@ -34,19 +34,35 @@
   {% endblock %}
 
   <div class="row">
-    <div class="col">
-      {% block cart_customer_information %}
-        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
-      {% endblock %}
-    </div>
-    <div class="col">
+    <div class="col-md-6">
       {% block cart_order_information %}
         {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/order_information.html.twig' %}
       {% endblock %}
     </div>
+    <div class="col-md-6">
+      {% block cart_customer_information %}
+        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
+      {% endblock %}
+    </div>
   </div>
 
   {% block cart_summary %}
     {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/cart_summary.html.twig' %}
   {% endblock %}
+
+  {% if cartView.cartSummary.cart_link is not empty %}
+    <div class="card" data-role="customer-information">
+      <h3 class="card-header">
+        <i class="material-icons">mail</i>
+        {{ 'Share this cart'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </h3>
+      <div class="card-body">
+        <p>{{ 'If you want to share this cart with a customer, send him the following link. After opening it, it will load the cart and all the customer information automatically. The customer can then finish the order in just a few clicks.'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Comment:
```suggestion
        <p>{{ 'Thanks to this link, your customer can open the cart on their device and check the contents of the order and their details before submitting payment.'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Reply:
@Julievrz 

`You can use the following link to share this cart. It will open the cart on your customer's device, allow to check its contents and finish the order if needed.`

Reply:
Ok then here's my suggestion (114 char. instead of 159): "Thanks to this link, your customer can open the cart, check its contents, and validate the order if it suits them."

Reply:
(No need to repeat the label "Share this cart" in the body message in my opinion.)